src/CocktailDetails/CocktailDetails.tsx

```diff
@@ -8,46 +8,55 @@ export interface CocktailDetailsProps {
 
 const CocktailDetails: React.FC<CocktailDetailsProps> = (props) => {
 	const [cocktailInfo, setCocktailInfo] = useState<RandomCocktail>({idDrink: '', strDrink: '', strInstructions: '', strDrinkThumb: ''})
-	const [error, setError] = useState<string>('');
```

Comment:
What is the benefit of removing <string> here?  I thought that it was a best practice in the resources we found?

Reply:
My 1/2 mentor told me this evening. She said for primitive data types like string, boolean, etc, we don't need to cast them in the useState statement. We usually only need to use it for more complex data like arrays and objects. @Melizzo

Reply:
TS infers on it's own for a primitive data type in useState's

Reply:
Ahhh, good to know, thank you.